packages/components/pagination/src/pagination.ts

```diff
@@ -137,6 +137,10 @@ export const paginationProps = buildProps({
     type: iconPropType,
     default: () => ArrowRight,
   },
+  /**
+   * @description whether Pagination size is teleported to body
+   */
+  teleported: Boolean,
```

Comment:
The default value written in the document is `true`, and the default value `true` should also be added here.

Reply:
Yes, i can change it, but isn't teleported already defaulted to true on select?

Reply:
Yes, in the `select` component, there is a `teleported` attribute with a default value of `true`.

But after you modify the code, the `select` component here receives the value passed from the `pagination` component. If the `teleported` attribute in the `pagination` component does not give a default value, Vue will convert its default value to `false`. At this time, the value received by the select component becomes `false`.

Reply:
Got it. Thanks.